health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/MultiSelectDropdown.js

```
@@ -0,0 +1,1020 @@
+        return [
+          ...state,
+          { code: action?.payload?.[1]?.code, name: action?.payload?.[1]?.name, propsData: action.payload },
+        ];
+        // Check if the item already exists to prevent duplication
+        if (state.some((e) => e.code === action.payload?.[1]?.code) || !action?.payload?.[1]?.type) {
+          return state; // Return state unchanged if item is already in queue
+        }
+        return [
+          ...state,
+          { code: action?.payload?.[1]?.code, name: action?.payload?.[1]?.name, propsData: action.payload },
+        ];
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)